django/core/checks/security/base.py

```diff
@@ -115,6 +122,21 @@
     id='security.E023',
 )
 
+W024 = Warning(
+    'You have not set the SECURE_FEATURE_POLICY setting. Without this, your site will not send a Feature-Policy '
+    'header. You should consider enabling this header to control which browser features can be used with your site.',
+    id='security.W024',
+)
+
+E025 = Error(
+    'You have set the SECURE_FEATURE_POLICY setting to an invalid value.',
+    hint=(
+        'Expected a dictionary with directives as keys and a list of strings of allowed origins as values. '
+        'Valid directives are: {}.'.format(', '.join(sorted(FEATURE_POLICY_DIRECTIVES)))
```

Comment:
The list is already sorted, so the sort is redundant.

Reply:
`FEATURE_POLICY_DIRECTIVES` is a set, not a list.